new-contract-interaction/src/agent.spec.js

```diff
@@ -72,22 +73,23 @@ describe('check agent configuration file', () => {
 
   describe('contracts key required', () => {
     const { contracts } = config;
-    expect(typeof (contracts)).toBe('object');
-    expect(contracts).not.toBe({});
+    Object.values(contracts).forEach((contract) => {
+      expect(typeof (contract)).toBe('object');
+      expect(contract).not.toBe({});
+    });
   });
 
   describe('contracts key values must be valid', () => {
     const { contracts } = config;
-    Object.keys(contracts).forEach((key) => {
+    Object.values(contracts).forEach((contract) => {
       const {
         thresholdBlockCount,
         thresholdTransactionCount,
         address,
         filteredAddresses,
         findingType,
         findingSeverity,
-      } = contracts[key];
-
+      } = contract.newContractEOA;
```

Comment:
`.newContractEOA` is referenced but doesn't exist in agent-config.json.

Reply:
oops sorry, looks like the last commit where I changed the agent-config.json file here didn't get pushed. Just pushed it up now!

Reply:
That makes more sense now.